src/validator/CreditCardVendorsReader.java

```diff
@@ -0,0 +1,58 @@
+package validator;
+
+import java.io.BufferedReader;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+
+public class CreditCardVendorsReader implements CreditCardVendorReading {
+	@Override
+	public ArrayList<CreditCardVendor> returnVendorArray() {
```

Comment:
1. I sugges to rename the method - e.g. "provideVendors", "initializeVendors"
    Avoid using type in method like "Array" , "String".
   The returned type is visible in signature.
2. Use List instead of ArrayList.